mock/generate-local-journal-non-test-activities.ts

```diff
@@ -0,0 +1,234 @@
+import * as enjoi from 'enjoi';
+import * as joi from '@hapi/joi';
+const journalSchema = require('@dvsa/mes-journal-schema/schema-examiner-work-schedule.json');
+import { writeFileSync } from 'fs';
+import { dirname, join } from 'path';
+import { getTodayAsIsoDate, getIsoDateDaysInFuture } from './mock-date-utils';
+
+const today = getTodayAsIsoDate();
+
+const localNtas = {
```

Comment:
`Ntas`? Non-test activities? If this is an abbreviation, maybe a longer variable name could be more descriptive.

Reply:
Renamed.